### ISSUE TITLE
Accessibility: Full Calendar Uses Aria-Hidden Inappropriately

### ISSUE THREAD

**Issue by kmmerc18** [fullcalendar/fullcalendar#7455]
### Reduced Test Case

https://codepen.io/Kimberly-Merchant/pen/abPyzYJ

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

Vue

### Bug Description

With a resource timeline like the one in my test case (taken straight from the demo versions on the FullCalendar website), some of our accessibility tests are failing.

Tool: aXe DevTools - https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd

If you scan the page using this automatic accessibility tracker, a number of issues come up. The one that's also being replicated in my company's code is "Certain ARIA roles must contain particular children"
![Screenshot 2023-09-14 at 1 16 38 PM](https://github.com/fullcalendar/fullcalendar/assets/60188830/5cc5bb85-7053-4dac-a1cd-83eec76e9378)
![Screenshot 2023-09-14 at 1 16 45 PM](https://github.com/fullcalendar/fullcalendar/assets/60188830/1a6e4c9b-c30e-4640-bc72-bd263ff3c74b)
![Screenshot 2023-09-14 at 1 16 51 PM](https://github.com/fullcalendar/fullcalendar/assets/60188830/54eae91b-7c0d-4f97-8692-ea14d4882767)

Either these elements within FullCalendar need different aria-roles so they are allowed to contain the aria-hidden attribute, or they need to not have that attribute.

**Comment by arshaw** [comment]
This whole mess of invalid aria tags on `<table>/<tr>`/etc elements is resolved in [v7.0.0-beta.4](https://github.com/fullcalendar/fullcalendar-workspace/releases/tag/v7.0.0-beta.4) because the DOM structure is leveraging aria/roles (see https://github.com/fullcalendar/fullcalendar/issues/6641#issuecomment-2579096269)
